code/game/objects/items/tajara.dm

```diff
@@ -146,47 +147,23 @@
 	else
 
 		var/adhomian_year = game_year + 1158
-
-		var/current_month = time2text(world.realtime, "Month")
+		var/current_month = text2num(time2text(world.realtime, "MM"))
 		var/current_day = text2num(time2text(world.realtime, "DD"))
 		var/adhomian_day
-		var/adhomian_month
+		var/adhomian_month = src.months[Ceiling(current_month/3)]
```

Comment:
Using modulo might be more appropriate.

Assuming a range of current_month from 1 to 12 this should work:
```suggestion
		var/adhomian_month = src.months[current_month - 1 % 4]
```

Reply:
Not equivalent. I think the correct one would be `(current_month - 1) % 3 + 1`, I considered doing a couple (micro-?)optimisations but this isn't hot code or anything, so I don't think it matters much.

Reply:
Oh I see, I didn't understand the calendar properly then. I thought they have only 4-month year, and instead it's a "regular" year where each month is 3 times as long. Carry on.